embedded-hal-async/README.md

```diff
@@ -36,20 +36,3 @@ Licensed under either of
 - MIT license ([LICENSE-MIT](LICENSE-MIT) or http://opensource.org/licenses/MIT)
 
 at your option.
-
-### Contribution
```

Comment:
I'm fine with removing the CoC notice in the subfolders but it might be good to keep the contribution section to avoid legal ambiguities.
(also below)